src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

Comment:
Good practice to add margin to all elements except of first/last child, depends on case 🔥 but use nesting here, in block above &:not(:last-child) {...}